### PR TITLE
Obc creation and deletion time

### PR DESCRIPTION
**Pull request by tiffanyn108**
OBC scale creation and deletion using Multi cloud Object Gateway
    * Creating up to max support number of OBCs, capture creation time in sec.
    * Deleting OBCs and capture deleting time in sec.

### CODE REVIEW

**Review by Avilir — APPROVED** [review]

File: ocs_ci/ocs/scale_noobaa_lib.py [hunk 1]
```diff
@@ -42,50 +45,50 @@ def construct_obc_creation_yaml_bulk_for_kube_job(no_of_obc, sc_name, namespace)
...
-                if status != constants.STATUS_BOUND:
-                    obc_not_bound_list.append(
-                        job_get_output["items"][i]["metadata"]["name"]
-                    )
+                status = job_get_output[i]["status"]["phase"]
+                log.info(f"obc {job_get_output[i]['metadata']['name']} status {status}")
+                if not status or status != constants.STATUS_BOUND:
+                    obc_not_bound_list.append(job_get_output[i]["metadata"]["name"])
+                    # Wait 20 secs to ensure the next obc on the list has status field populated
+                    time.sleep(20)
```
Comment (ebenahar) [inline ocs_ci/ocs/scale_noobaa_lib.py:78]: Please change to sampling the existence of the next obc on the list to be sure it is populated

**Comment by ramkiperiy** [comment]
@tiffanyn108 are we able to solve the OBC creation timeout issue ? i.e. status getting updated ?

File: ocs_ci/ocs/scale_noobaa_lib.py (hunk 1 above)
Comment (tiffanyn108) [reply ocs_ci/ocs/scale_noobaa_lib.py:78]: will address this in next PR.

**Comment by tiffanyn108** [comment]
@ramkiperiy Fixed tox error.  All checks are passed now.

**Review by ramkiperiy — APPROVED** [review]
LGTM

**Review by Avilir — APPROVED** [review]
LGTM

**Review by pintojoy — APPROVED** [review]
LGTM

**Pull request merged**